easybuild/easyconfigs/g/Go/Go-1.20.4.eb

```diff
@@ -0,0 +1,31 @@
+# Update to 1.20.4
+# Jordi Camps / CNAG
+# Sebastien Moretti / SIB
+
+easyblock = 'Tarball'
+
+name = 'Go'
+version = "1.20.4"
+
+homepage = 'https://www.golang.org'
+description = """Go is an open source programming language that makes it easy to build
+ simple, reliable, and efficient software."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://storage.googleapis.com/golang/']
+
+local_archs = {'aarch64': 'arm64', 'x86_64': 'amd64'}
+sources = ['go%%(version)s.linux-%s.tar.gz' % local_archs[ARCH]]
+checksums = ['698ef3243972a51ddb4028e4a1ac63dc6d60821bf18e59a807e051fee0a385bd']
```

Comment:
It was like that in the other Go eb files e.g. Go-1.18.3.eb

I will add the checksum for aarch64 in `Go-1.20.4.eb`

Reply:
Some tests failed because
`Checksums missing for one or more sources/patches in Go-1.20.4.eb: found 1 sources + 0 patches vs 2 checksums`

I think this is why only one checksum was provided.
I will try tomorrow with a condition per architecture.

Reply:
like in https://github.com/easybuilders/easybuild-easyconfigs/pull/16946/files#diff-77e2d80bf0508278d1f943bcbca2559f33c5e89eb2a15df57ed16f4274fd024a

Reply:
see Java for examples: https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/j/Java/Java-17.0.6.eb